src/seedu/addressbook/data/person/Address.java

```diff
@@ -52,7 +52,17 @@ public int hashCode() {
         return value.hashCode();
     }
 
+    @Override
+    public String getPrintableString() {
+        if (isPrivate) {
```

Comment:
This implementation causes `getPrintableString()` to handle more than one responsibility, recall our tutorial on *SRP*.
Perhaps another higher level class should handle what to display based on the `isPrivate` flag, say at `ReadOnlyPerson.java`?